apps/mobile/tsconfig.json

```diff
@@ -1,5 +1,8 @@
 {
   "extends": "../../tsconfig",
+  "compilerOptions": {
+    "baseUrl": "../.."
+  },
```

Comment:
We set this so that Expo / TypeScript knows to resolve paths (set in root tsconfig.json) from root of this workspace.

Reply:
ok gotcha